### PR TITLE
#5153 Add IT tests for files of settings dialog

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5153)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/support/customCommands.ts [hunk 1]
```diff
@@ -291,16 +303,25 @@ Cypress.Commands.add(
...
     columnName: string,
     columnValue?: string,
+    rowFilter?: (row: JQuery<HTMLTableElement>) => boolean,
   ) => {
     const header = $table.find('thead tr').find(`th:contains("${columnName}")`);
     const index = $table.find('thead tr th').index(header);
 
+    let tableRows = null;
+    if (rowFilter) {
+      tableRows = $table.find('tbody tr').filter(function (_, element) {
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/customCommands.ts:313]: @saivirtue could we convert this to an arrow function for the sake of consistency :) I'm guessing using `function` and `arrow` function would work the same as we don't use `this` here?

File: ohara-manager/client/cypress/it/navigator/settings.test.ts [hunk 2]
```diff
@@ -641,6 +671,355 @@ describe('Navigator', () => {
...
+            $table,
+            'Actions',
+            undefined,
+            (row) => row.has(`td:contains("${fakeJar.name}")`).length > 0,
+          ).within(() => {
+            cy.findByTitle('Delete file').click();
+          });
+        });
+        cy.findByTestId('confirm-button-DELETE').click();
+        cy.wait(500);
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/navigator/settings.test.ts:923]: A comment for this `wait` would clarify the indent :)

File: ohara-manager/client/cypress/support/customCommands.ts (hunk 1 above)
Comment (saivirtue) [reply ohara-manager/client/cypress/support/customCommands.ts:313]: Sure, let me check this.

File: ohara-manager/client/cypress/it/navigator/settings.test.ts (hunk 2 above)
Comment (saivirtue) [reply ohara-manager/client/cypress/it/navigator/settings.test.ts:923]: wait a second :)

**Comment by saivirtue** [comment]
retry manager-it

**Pull request merged**